saleor/core/utils/url.py

```diff
@@ -13,12 +13,8 @@ def validate_storefront_url(url):
     """
     try:
         parsed_url = urlparse(url)
+        domain, _ = split_domain_port(parsed_url.netloc)
     except ValueError as error:
         raise ValidationError({"redirectUrl": str(error)})
-    if not validate_host(parsed_url.netloc, settings.ALLOWED_STOREFRONT_HOSTS):
-        raise ValidationError(
-            {
-                "redirectUrl": "%s this is not valid storefront address."
-                % parsed_url.netloc
-            }
-        )
+    if not validate_host(domain, settings.ALLOWED_STOREFRONT_HOSTS):
```

Comment:
We should change the `ALLOWED_STOREFRONT_HOSTS` setting name to something more generic as it is used with either storefronts or dashboard. Maybe `ALLOWED_CLIENT_HOSTS`?